src/common/dsp/modulators/MSEGModulationHelper.h

```diff
@@ -19,7 +19,7 @@ struct EvaluatorState
     }
     int lastEval = -1;
     float lastOutput = 0;
-    float msegState[6] = {0};
+    float msegState[n_lfos_voice] = {0};
```

Comment:
This one is wrong. That 6 just happens to be the total number of state elements i need to evaluate the mseg forward for a particular voice. If you want you can make it

```
static constexpr int n_msegStateElements = 6;
msegState[n_msegStateElements];
```
but don't merge this change since it is a bit deceiving.

Reply:
(you could also imagine calling his data structure 'msegStateRegisters')